core-web/libs/sdk/react/src/lib/utils/utils.ts

```diff
@@ -96,7 +96,10 @@ export const getContainersData = (
 ) => {
     const { identifier, uuid } = containerRef;
 
-    const { containerStructures } = containers[identifier];
+    const { containerStructures, container } = containers[identifier];
+
+    // Get the variant id
+    const { variantId } = container?.parentPermissionable || {};
```

Comment:
Is variantId a string or number? Can we a the type here?

Reply:
The parentPermissionable is a `Record<string, string>`

Here: https://github.com/dotCMS/core/blob/48547c9d4b4b10cdbd80c98d92554dbb4bdec0b6/core-web/libs/sdk/react/src/lib/components/PageProvider/PageProvider.tsx#L17